### PR TITLE
Add i18n to js messages

### PR DESCRIPTION
**Pull request by int-y1**
Translate these messages:

- error when voting, e.g. attempt to self-vote
- warning popup when hiding comment
- error when hiding comment
- "updated" message after editing a comment
- `/problem/aplusb/test_data` > Checker > select Floats > hover over input box
- error when closing/opening ticket

### CODE REVIEW

File: templates/comments/media-js.html [hunk 1]
```diff
@@ -169,9 +169,9 @@
                                 if (window.add_code_copy_buttons)
                                     window.add_code_copy_buttons($area);
                                 var $edits = $comment.find('.comment-edits').first();
-                                $edits.text('updated');
+                                $edits.text('{{ _('updated') }}');
                             }).fail(function () {
-                                console.log('Failed to update comment:' + id);
+                                console.log('Failed to update comment: ' + id);
```
Comment (Ninjaclasher) [inline templates/comments/media-js.html:174]: Should this receive the same treatment?

Comment (int-y1) [reply templates/comments/media-js.html:174]: I left it out because the user never sees `console.log`'s output.

Comment (Ninjaclasher) [reply templates/comments/media-js.html:174]: I think we should change this to an `alert`.

Comment (int-y1) [reply templates/comments/media-js.html]: Done. The user will see `Failed to update comment body.`

File: templates/problem/data.html [hunk 2]
```diff
@@ -1,15 +1,10 @@
 {% extends "base.html" %}
 
 {% block js_media %}
-    <script type="text/javascript">
-        window.valid_files = {{valid_files_json}};
-
-        $(function () {
-        });
```
Comment (Xyene) [inline templates/problem/data.html:8]: A+

**Review by Xyene — APPROVED** [review]
LGTM, thanks!

**Pull request merged**